datashape/discovery.py

```diff
@@ -405,3 +405,19 @@ def descendents(d, x):
         children -= desc
         desc.update(children)
     return desc
+
+
+Mock = None
+try:
+    from unittest.mock import Mock
+except ImportError:
+    try:
+        from mock import Mock
+    except ImportError:
+        pass
+
+if Mock is not None:
+    @dispatch(Mock)
```

Comment:
i really don't like breaking this, but i also don't think this was the right way to go in the first place